src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/FirstHop.tsx

```diff
@@ -113,7 +61,8 @@ export const FirstHop = ({
 
   // TODO: use `isApprovalNeeded === undefined` here to display placeholder loading during initial approval check
   const {
-    isApprovalNeeded,
+    wasApprovalNeeded,
```

Comment:
This is quite confusing especially when passing one instead of the other - do we want to keep this as `isApprovalNeeded` and rely on the `approvalTxId` to infer whether or not an approval was originally needed?
i.e `wasApprovalNeeded = isApprovalNeeded || approvalTxId` here, but don't change the actual signature

Reply:
🧠 big brain ideas, will implement this

Reply:
Tackled in https://github.com/shapeshift/web/pull/5623 @woodenfurniture